crowbar_framework/lib/crowbar/backup/base.rb

```diff
@@ -42,8 +42,16 @@ def restore_files
               "/root/.chef"
             ],
             [
-              "data",
-              "/var/lib/crowbar"
+              "data/backup",
+              "/var/lib/crowbar/backup"
+            ],
+            [
+              "data/cache",
+              "/var/lib/crowbar/cache"
+            ],
+            [
+              "data/config",
+              "/var/lib/crowbar/config"
```

Comment:
Yes it's missing but wanted in this case, we don't need this files because while restore we delete this files.